pkg/controller/controlplane/valuesprovider.go

```diff
@@ -116,6 +116,17 @@ var ccmShootChart = &chart.Chart{
 	},
 }
 
+var udpEgressDummyChart = &chart.Chart{
+	Name: "udp-egress-dummy",
+	Path: filepath.Join(azure.InternalChartsPath, "udp-egress-dummy"),
```

Comment:
This path looks wrong. Why have we moved the things into a `controlplane-shoot` subfolder in the first place?